### DISCUSSION TITLE
add rebrand assets to craft cloud

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cloud#10] [Feature Requests & Enhancements]
Rebrand assets don't seem to be showing.

Login Page Logo
Site Icon

These are set up locally, a part of the repo, and Craft outputs them in storage/rebrand.

**Comment by timkelty** [comment]
Thanks @matthewstick, confirmed this is a bug – will get fixed ASAP.

**Comment by timkelty** [comment]
This is addressed by the release of Craft 4.10/5.2, with the addition of `CRAFT_REBRAND_PATH`: https://craftcms.com/docs/5.x/reference/config/bootstrap.html#craft-rebrand-path

If they are in your repo at `storage/rebrand`, you'll want to add the following environment variable:
```
CRAFT_REBRAND_PATH=@root/storage/rebrand
```
